src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -18,13 +18,16 @@
 
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
-import seedu.address.model.person.Person;
+import seedu.address.model.person.Patient;
 import seedu.address.model.person.exceptions.DuplicatePersonException;
-import seedu.address.testutil.PersonBuilder;
+import seedu.address.testutil.PatientBuilder;
 
+/**
+ * Specialized to {@code AddressBook<Patient>}.
+ */
```

Comment:
I don't think so? I think this should be a class that tests for all versions of addressbook? because this would correspond to the main class `AddressBook.java`, which is for both doctors and patients right

Reply:
Will remove comment, but in order to test doctors too will require another AddressBook<Doctor> as a parameter in the future

Reply:
yup i think so